src/mame/galaxian/galaxian.cpp

```diff
@@ -10682,7 +10650,24 @@ ROM_START( omegab )
 ROM_END
 
 
-ROM_START( warofbug )  // Shows 20c as the base of currency.  I know of no US games that used dimes - unless it's another country?
+/* Galaxian hardware mods for War of the Bugs warofbug/warofbugg
+
+(WotB using daughterboard that plugs into Z80 socket - has a socketed PAL, two other 20-pin ICs,
+a 16-pin IC (all with their markings sanded off) and five EPROMs)
+
+On the Galaxian board:
+
+2B, 74LS366 - cut pin 15 OR pin 1 (this disables the stars)
+8E, 74LS139 - cut pin 11 only, join the stub left ON THE CHIP to pin 13 on the chip
+
+Cut the track on the bottom of the board going to pin 21 of IC 1K (2716 EPROM).
+
+Cut the track on the bottom of the board going to pin 21 of IC 1H (2716 EPROM).
+
+Join pins 21 of IC's 1H and 1K together and connect both to +5 volts (although
+I've seen one set of docs saying to connect to pin 1 of ic 2N (7408) instead). */
+
+ROM_START( warofbug )  // Shows 20c as the base of currency. I know of no US games that used dimes - unless it's another country?
```

Comment:
Probably an Australian set – 20c was the standard price for one game in the ’80s.